.changeset/chilled-weeks-occur.md

```diff
@@ -0,0 +1,7 @@
+---
+"@crossmint/client-sdk-react-native-ui": minor
+"@crossmint/client-sdk-react-ui": minor
```

Comment:
We're doing patch versions in react libraries for all the ncs updates. We also do a separate changeset file for each library to manage the changelogs differently. But if it's the same update for all, no worries